examples/demo.rb

```diff
@@ -295,7 +295,7 @@
   # format is a fixed-size image, so I don't need to specify a size.
   puts 'Adding logo image...'
   logo = Image.read('logo:').first
-  if /GraphicsMagick/.match Magick_version
+  if Magick_version =~ /GraphicsMagick/
```

Comment:
@Watson1978 can we not use `.match?` for these? For some reason I thought that would be more performant.

```diff
- if /GraphicsMagick/.match Magick_version
+ if Magick_version.match?(/GraphicsMagick/)
```

Reply:
Because `String#match?` was introduced at Ruby 2.4.0
https://github.com/ruby/ruby/blob/17840baed96a296c0e0678a8c686a228859fa7ac/NEWS?plain=1#L162

So, if we can drop Ruby 2.3 support, it can change it with suggested code...